test/EthLightClient.js

```diff
@@ -0,0 +1,150 @@
+const { expect } = require("chai");
+const { ethers, upgrades } = require("hardhat");
+const block1 = require('./proof1.json');
+const block2 = require('./proof2.json');
+
+function hexToBytes(hex) {
+    for (var bytes = [], c = 0; c < hex.length; c += 2)
```

Comment:
Should var be removed everywhere?

Reply:
Yes, its a good idea to keep it consistent. I grabbed this hexToBytes function out of the bridge.hmy.js without examining it super carefully. Will change it there too.